src/build.py

```diff
@@ -1785,6 +1800,9 @@ def SplitComma(arg):
     parser.add_argument(
         '--clobber', dest='clobber', default=False, action='store_true',
         help="Delete working directories, forcing a clean build")
+    parser.add_argument(
+        '--extraopt', dest='extraopt', default=False, action='store_true',
```

Comment:
This reads to be as "extra options" so I expected to be able to pass more options this way.   I don't have any great alternatives though.   We could just called it `--enable-lto` I guess?

Reply:
I also found `extraopt` slightly confusing. Maybe `enable-lto` is less confusing even though it also disables assertions. I don't have an idea for a better name.

Reply:
`enable-lto` seems fine to me. If we ever change it so that assertions are selected differently from lto we can update to whatever makes sense.

Reply:
I went with `use-lto` to match the use_sysroot option (and to provide just a little extra disambiguation between using LTO in the build and enabling LLVM's LTO target which builds libLTO)